packages/fxa-auth-db-mysql/lib/db/schema/patch-114-115.sql

```diff
@@ -0,0 +1,54 @@
+--
+-- This migration introduces a table to track sent emails related to
+-- subscriptions.
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('114');
+
+CREATE TABLE IF NOT EXISTS sentSubscriptionEmails (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  uid BINARY(16) NOT NULL,
+  emailType VARCHAR(60) NOT NULL,
+  subscriptionId VARCHAR(255) NULL,
+  planId VARCHAR(255) NULL,
+  sentAt BIGINT UNSIGNED NOT NULL,
+  INDEX sentSubscriptionEmailsIndex(uid, emailType, subscriptionId, planId)
```

Comment:
If subscriptionId/planId are replaced with a JSON column of say `params` or something, indexing it perhaps isn't needed. I'm not sure what the utility would be in finding emails across all users sent for different sub/plan id's.

Reply:
If we don't index anything in the JSON column, then MySQL would need to scan the results for uid+emailType.  That probably will be ok?  As a FxA user and VPN subscriber, the volume of emails from FxA is pretty low.

Reply:
Yea, I don't think we need to index the params.